README.md

```diff
@@ -45,6 +45,29 @@ Run `pcvalidate --help` for the available command line flags.
 
 The tool returns 0 in case of successful validation, 1 otherwise.
 
+## Easy validation with Docker
```

Comment:
Why "easy"? :smile:

Reply:
@libremente because it's easy with Docker :) But if you prefer I'll remove it..